src/clinical/donor-repo.ts

```diff
@@ -117,6 +121,25 @@ export const donorDao: DonorRepository = {
     return F(mapped);
   },
 
+  // Like findByProgramId, but DocQuery asks mongo to return PoJo without docIds for faster fetch
+  async findByProgramIdOmitMongoDocId(
+    programId: string,
+    projection?: Partial<Record<DONOR_FIELDS, number>>,
+  ): Promise<DeepReadonly<Donor[]>> {
+    const result = await DonorModel.find(
+      {
+        [DONOR_FIELDS.PROGRAM_ID]: programId,
+      },
+      projection,
+      { sort: { [DONOR_FIELDS.DONOR_ID]: 1 } },
```

Comment:
why are we sorting here ? just for my information

Reply:
Not sure. This was copy pasta from `findByProgramId` so it was already their. I think it's related to the DonorSchema.index